File: labs/lab1/app/src/main/java/org/example/Product.java
package org.example;

class Product {
	public static int unique_id = 0;

	private int id;
	private String description;
	private float price;

	public Product (String description, float price) {
		this.id = unique_id;
		unique_id++;
		this.description = description;
		this.price = price;
	}
	
	@Override
	public String toString(){
		// TODO: 
		return "";
	}
}

File: labs/lab06/src/main/scala/example/Product.scala
package example

object Product {
  // Takes in the comma seperated input from Product and creates an dataclass
  def parse(input: String): Product = {
    var i = input.split(",").map(_.trim)

    return Product(
      i(0).toInt, 
      i(1), 
      i(2).toDouble)
  }
}

// productId, description, price
case class Product(
  private val productId: Int, 
  private val description: String, 
  private val price: Double) {
    override def toString: String =
      s"($productId, $description, $price)"
  }
